### PR TITLE
Building the new UI

### PR DESCRIPTION
**Pull request by the94air**
This PR should contain UI related changes. Will be adding commits here consistently for previews.

_P.S. Don't merge this PR yet_

### CODE REVIEW

**Comment by Rademenes16** [comment]
these bots are a bad joke lol

**Comment by lostdesign** [comment]
@Rademenes16 yeah, Vercel wants us to pay - but we are in the process of getting our open source project validated so that we can finally deploy again. Meanwhile just ignore verbal and check the netlify builds as they still work.

**Comment by OmarUsman** [comment]
Looking amazing :)

File: components/Logo.vue [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
-    opacity: 0;
+export default {
+  components: { Logo },
+  mounted() {
+    this.sparkle();
+  },
+  methods: {
+    sparkle() {
+      const svg = this.$refs.logo;
+      const paths = this.$refs.logo.childNodes;
```
Comment (kgnfth) [inline components/Logo.vue:16]: should it not be like this `const paths = svg.childNodes;` ?

Comment (the94air) [reply components/Logo.vue:16]: Does it make a difference? :thinking:

Comment (kgnfth) [reply components/Logo.vue:16]: well, otherwise i get error saying `Expected an assignment or function call and instead saw an expression`

Comment (kgnfth) [reply components/Logo.vue:16]: you are defining svg but you are not using it anywhere

**Pull request closed without merging**